host/4/bullseye/amd64/python/python37/python37-composite.template

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.7-slim-bullseye
```

Comment:
@balag0 / @vrdmr  - any reason to use buster instead for bullseye?

Reply:
Probably at that time due to - https://github.com/Azure/azure-functions-docker/issues/481

Reply:
Yeah - thanks @Balag0 for pointing this out. This was due to buster not being available. This should be good to go now.